bundle/manifests/compliance.openshift.io_tailoredprofiles.yaml

```diff
@@ -97,13 +107,18 @@ spec:
                   description: RuleReferenceSpec specifies a rule to be selected/deselected,
                     as well as the reason why
                   properties:
+                    kind:
+                      description: Type of rule, either "Rule" or "CustomRule". "Rule"
+                        is the default if not specified.
```

Comment:
Same comment here as the `disabledRules` above. Do we need to support `CustomRules` here initially? Or can we add that in later?